### PR TITLE
cli stake tests: Increase stake amounts to handle raising minimum delegation

### PR DESCRIPTION
**Pull request by brooksprumo**
#### Problem

The CLI stake tests fail once the stake minimum delegation is raised. See #24603 for more context.

#### Summary of Changes

- Increase the stake amounts
- Remove solana_logger::setup() from tests to be less noisy by default

### CODE REVIEW

File: cli/tests/stake.rs [hunk 1]
```diff
@@ -1218,17 +1264,29 @@ fn test_stake_set_lockup() {
...
+    )
+    .unwrap();
+    check_balance!(1_000_000_000_000, &rpc_client, &offline_pubkey);
 
     // Create stake account, identity is authority
     let minimum_stake_balance = rpc_client
         .get_minimum_balance_for_rent_exemption(StakeState::size_of())
-        .unwrap();
+        .unwrap()
+        + 10_000_000_000;
```
Comment (joncinque) [inline cli/tests/stake.rs:1289]: nittiest of nits: perhaps this variable name should get changed since it isn't really the minimum stake balance

**Review by joncinque — APPROVED** [review]
Just an uber-nit the rest is good to go!

**Comment by CriesofCarrots** [comment]
Fwiw, I kind of like the logging as reassurance that it's some tests with lots of confirmation, instead of a hang. But maybe that's just vestigial from the days when cargo test was short enough that it made some sense to watch its progress...

**Comment by brooksprumo** [comment]
> Fwiw, I kind of like the logging as reassurance that it's some tests with lots of confirmation, instead of a hang. But maybe that's just vestigial from the days when cargo test was short enough that it made some sense to watch its progress...

Ok, I'll re-add the logging.

File: cli/tests/stake.rs (hunk 1 above)
Comment (joncinque) [reply cli/tests/stake.rs:1289]: Sounds good!

Comment (brooksprumo) [reply cli/tests/stake.rs]: Update: It's `stake_balance` now 😁

**Comment by brooksprumo** [comment]
@joncinque Oops, I see you already approved the changes. No need to re-review; my mistake!

**Pull request merged**